### PR TITLE
Update libpng version to >=1.6.28

### PR DESCRIPTION
**Pull request by agoodm**
Fix for #103

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ requirements:
     - pytz
 #    - py2cairo  # [linux and py2k]
     - tornado
-    - libpng >=1.6.23,<1.7
+    - libpng >=1.6.28,<1.7
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Please change it below too.

Comment (agoodm) [inline recipe/meta.yaml]: Done.

**Comment by astrofrog** [comment]
@ocefpaf - what is the license warning about? (looks like nothing was changed here relating to the license)

**Comment by ocefpaf** [comment]
Someone changed the linter the be pedantic and raise warning when the word
'license' is  used :-(

Just ignore it as and merge this if  you are ok.  There is no need to
satisfy obsessive compulsive disorder of the linter.

**Pull request merged**